flw/src/main/resources/flw.properties

```diff
@@ -0,0 +1,7 @@
+#what time of day should the process of generating the targetFile start
```

Comment:
I just copied your code from the cdr scheduling.  Do you mean I should change the comment or remove the ability to set the time?